testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1581,6 +1582,35 @@ private Optional<ZonedDateTime> getMaterializedViewLastFreshTime(String material
         return Optional.ofNullable(lastFreshTime);
     }
 
+    @Test
+    public void testColumnCommentMaterializedView()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW));
+        skipTestUnless(hasBehavior(SUPPORTS_COMMENT_ON_MATERIALIZED_VIEW));
+
+        String catalog = getSession().getCatalog().orElseThrow();
+        String schema = getSession().getSchema().orElseThrow();
+
+        QualifiedObjectName view = new QualifiedObjectName(catalog, schema, "test_materialized_view_" + randomNameSuffix());
+
+        createTestingMaterializedView(view, Optional.empty());
+
+        assertUpdate("COMMENT ON COLUMN " + view.getObjectName() + ".name IS 'new comment'");
+        assertThat(getColumnComment(view.getObjectName(), "name")).isEqualTo("new comment");
+
+        // comment deleted
+        assertUpdate("COMMENT ON COLUMN " + view.getObjectName() + ".name IS NULL");
+        assertThat(getColumnComment(view.getObjectName(), "name")).isEqualTo(null);
+
+        // comment set to non-empty value before verifying setting empty comment
+        assertUpdate("COMMENT ON COLUMN " + view.getObjectName() + ".name IS 'updated comment'");
+        assertThat(getColumnComment(view.getObjectName(), "name")).isEqualTo("updated comment");
+
+        // comment set to empty or deleted
+        assertUpdate("COMMENT ON COLUMN " + view.getObjectName() + ".name IS ''");
+        assertThat(getColumnComment(view.getObjectName(), "name")).isIn("", null); // Some storages do not preserve empty comment
```

Comment:
> Some storages do not preserve empty

Is it copy-paste left over? Or do we already have such connector/metastore?

Reply:
it's left over, removed